.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["esbenp.prettier-vscode", "dbaeumer.vscode-eslint"]
```

Comment:
This should use tabs.

Also add:
- `tamasfe.even-better-toml`
- `github.vscode-pull-request-github`
- `codezombiech.gitignore`
- `eamodio.gitlens`
- `christian-kohler.npm-intellisense`
- `christian-kohler.path-intellisense`